dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/config/DataSourceConfig.java

```diff
@@ -102,8 +95,8 @@ public JdbcTemplate readOnlyJdbcTemplate(@Qualifier("dataSource") DataSource dat
     return jdbcTemplate;
   }
 
-  @Bean("actualDataSource")
-  public DataSource actualDataSource(
+  public static DataSource getActualDataSource(
+      DhisConfigurationProvider dhisConfig,
```

Comment:
I'm not sure if we want to have separate connection pool for each dataSource for same database.
There would be confusion to the connection pool settings in `dhis.conf` file. 
For example `CONNECTION_POOL_MAX_SIZE` or `CONNECTION_POOL_MAX_IDLE_TIME` , etc.

Reply:
yes those values should be then fine tuned